v8_create.go

```diff
@@ -145,14 +145,20 @@ func (ctx *Context) createWithTags(val reflect.Value, tags []string) (*Value, er
 		keys := val.MapKeys()
 		sort.Sort(stringKeys(keys))
 		for _, key := range keys {
-			v, err := ctx.create(val.MapIndex(key))
+			toCreate := val.MapIndex(key)
+			if toCreate.Kind() == reflect.Interface {
+				toCreate = toCreate.Elem()
```

Comment:
The tests are great for verifying that pre-existing valuePtrTypes are released, but it doesn't check for this interface de-referencing.  I think it's safer to have `.create` return a boolean that indicates whether a value was allocated.